snooty/rstspec.toml

```diff
@@ -463,6 +463,45 @@ example = """.. code:: ${1:language}
    ${0:code content}
 """
 
+[directive.io-code-block]
+help = """A block of code and its corresponding output to show the user."""
+content_type = "block"
+options.caption = "string"
+options.copyable = "boolean"
+options.class = "string"
+options.source = "uri"
+example = """.. io-code-block::
```

Comment:
We've had a request to comment out the example for these directives so it won't shop up in the snippet view when folks are working in VSCode, since we'll be testing this in production and the content folks don't want to accidentally use it before they've had a chance to UAT.

TL:DR, please comment out the examples; we'll restore them in a couple of weeks once we've got the final LGTM.